Sources/NSIndexPath+Sweetness.swift

```diff
@@ -2,21 +2,28 @@ import UIKit
 
 public extension NSIndexPath {
     public enum Direction {
-        case Forward
-        case Backward
         case Same
+        case Before
+        case Ahead
     }
 
-    public func compareDirection(indexPath: NSIndexPath) -> Direction {
-        let current = self.row * self.section
-        let coming = indexPath.row * indexPath.section
-
-        if current == coming {
+    private func comparePosition(ownRow ownRow: Int, otherRow: Int) -> Direction {
```

Comment:
`ownRow` sounds kind of weird. I think is fine just by not having a parameter name here.